explainer.md

```diff
@@ -336,6 +325,62 @@ if(frameOfRef.bounds) {
 }
 ```
 
+### Mirroring
```

Comment:
Should this go in the advanced functionality section?  On mobile devices, this won't be particularly useful and on Windows Mixed Reality headsets, there is automatic mirroring in the Mixed Reality Portal.